monkey/monkey_island/cc/ui/src/components/ui-components/XDataGrid.js

```diff
@@ -72,10 +75,27 @@ const prepareColsWidth = (columns, columnWidth, setColWidth) => {
   return updatedColumns;
 }
 
+const prepareColsCustomTooltip = (columns) => {
+   let updatedColumns = _.cloneDeep(columns) || [];
+   updatedColumns?.forEach((col)=>{
```

Comment:
Is the "?" needed here, or does the previous line ensure that the variable has a value?

Reply:
nothing is ensured